charts/kyverno/values.yaml

```diff
@@ -258,7 +258,8 @@ installCRDs: true
 # This is useful on clusters using Calico and/or native k8s network
 # policies in a default-deny setup.
 networkPolicy:
-  enabled: false
+  enabled: true
+  allowOCIAccess: false
```

Comment:
Can we add some use facing docs for the values in this object in the chart readme?

Reply:
Should we change the variable name to something more generic? Suggestions?

Reply:
Probably something like `allowAllOutgoing` or `openEgress` or something since it is not really opening it specifically for registries but for all outgoing traffic. Being able to access registries is a result. 

Regardless, we would need to update the table at https://github.com/kyverno/kyverno/tree/main/charts/kyverno#configuration

Reply:
Hmm, sounds good